part-i.md

```diff
@@ -459,12 +459,12 @@ data Bike = Bike
 {% tabs %}
 {% tab title="TypeScript \(Direct\)" %}
 ```typescript
-const makeGear = (big, little): GearRatio => { big, little }
+const makeGear = (big, little): GearRatio => ({ big, little })
```

Comment:
Without the parentheses, those curly braces denote a statement and not an expression. In other words, `makeGear` was returning `void` instead of `GearRatio`.